src/Helper.js

```diff
@@ -4,14 +4,15 @@
 
 const Webdriver = require('./Webdriver');
 const Features = require('./Features');
+const async = require('async');
```

Comment:
Please rename that to Async according to the naming of the other imported modules here :)